### DISCUSSION TITLE
Building the svelte project fails in my mono-repo setup.

### DISCUSSION THREAD

**Discussion by elvis-ndubuisi** [rotimi-best/classroomio#48] [Q&A] [answered]
Hello everyone. I'm currently migrating the project to a Monorepo workspace. The Svelte app isn't build command fails with this logs>
![Screenshot 2023-11-25 110405](https://github.com/rotimi-best/classroomio/assets/24261142/665db4e7-1c6b-4af9-96cd-f56b8102868a)

Is it an issue from my end?

Also, i belive this monorepo comes with a log of breaking changes

**Comment by rotimi-best** [comment]
@elvis-ndubuisi can you please share how you fixed this

**Reply by elvis-ndubuisi** [reply]
Okay

**Reply by elvis-ndubuisi** [reply, marked as answer]
![Screenshot 2023-12-13 213522](https://github.com/rotimi-best/classroomio/assets/24261142/3cc46342-638b-4688-9e5a-98a1e54f2c0c)

This was the step i took when the dayjs issue came up in mine. Here is the article on that

https://elvisndubuisi.substack.com/p/a-journey-into-converting-an-open

**Reply by rotimi-best** [reply]
Thank you
